config/kubernetes/sig-docs/teams.yaml

```diff
@@ -339,6 +341,7 @@ teams:
     - gochist
     - hanjiayao
     - huynguyennovem
+    - ianychoi
```

Comment:
This is based on https://github.com/kubernetes/website/blob/8f7f707d6b9ab27b393638134729f5ced94f2818/OWNERS_ALIASES#L149 (AIUI)

Reply:
@ysyukr Please limit this PR only to yourself. Other members can add themselves to various groups as need be.

Reply:
@zacharysarah I agree with you. I have updated this PR to be registered only with me.